package.json

```diff
@@ -36,7 +37,8 @@
     "postgrator-cli": "^4.0.0",
     "prettier": "^2.4.1",
     "sinon": "^12.0.1",
-    "tap": "^15.1.1"
+    "tap": "^15.1.1",
+    "typescript": "^4.4.4"
```

Comment:
`tap` and `typescript` packages should be moved into `devDependencies`. Unless I'm missing something, they won't be shipped in any case we're about to build. Could you move them into `devDependencies`, please?